test/test_svd.cc

```diff
@@ -57,6 +58,26 @@ void test_svd_work( Params& params, bool run )
     params.ortho_V();
     params.error.name( "S - Sref" );
     params.error2.name( "Backward" );
+    if (timer_level >= 2) {
+        params.time2();
+        params.time3();
+        params.time4();
+        params.time5();
+        params.time6();
+        params.time7();
+        params.time8();
+        params.time9();
+        params.time10();
+        params.time2.name( "geqrf (s)" );
+        params.time3.name( "gelqf (s)" );
```

Comment:
`geqrf` and `gelqf` are mutually exclusive (QR happens if m >> n, LQ happens if m << n), so possibly we could have one timer here instead of two timers.

Similarly `unmqr` (for U) and `unmlq` (for V) below are mutually exclusive. They apply the effects of these `geqrf` or `gelqf`.

Thoughts? It's fine to leave as-is, just 2 of the columns will always be zero. If the matrix is "close" to square (1/threshold*n < m < threshold*n), then all 4 columns will be zero.

Reply:
I believe it makes sense to leave it such that you have the additional 2 columns that show up as zero. In my opinion it gives the full picture of what is occurring at each piece of the routine.